src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -1564,6 +1564,42 @@ def move_instrumentlocations(tool):
     logger.info("Move Instrument Locations [DONE]")
 
 
+@upgradestep(product, version)
+def move_samplecontainers(tool):
+    """Move sample containers to senaite setup folder
+    """
+
+    # run required import steps
+    tool.runImportStepFromProfile(profile, "typeinfo")
+    tool.runImportStepFromProfile(profile, "workflow")
+
+    # get the old container
+    origin = api.get_setup().get("sample_containers")
+    if not origin:
+        # old container is already gone
+        return
+
+    # get the destination container
+    destination = get_setup_folder("sample_containers")
```

Comment:
Would you mind to change this to `samplecontainers` to match the naming scheme we used for the other setup folders?

Reply:
oh, sure. i was think about of renaming folder for this content type, but decided to wait for your consultation

Reply:
Thanks!